objects/oaudiomanager/Create_0.gml

```diff
@@ -0,0 +1,44 @@
+/// @description Insert description here
+// You can write your code in this editor
+audio_group_stop_all(audiogroup_mus)
+audio_group_load(audiogroup_mus);
+audio_group_load(audiogroup_sfx);
+
+//switch room 
```

Comment:
There is no need for these comments since this code is in Step. Keep the code less cluttered.

Reply:
got it, forgot to remove it